.github/workflows/build_lint.yml

```diff
@@ -32,15 +32,15 @@ jobs:
           - os: ubuntu
             python-version: "3.7"
           - os: ubuntu
-            python-version: pypy3
+            python-version: 'pypy-3.8'
```

Comment:
Was about to mention this minor nit: double quote so that it is consistent with the other quotes usage, but nvm.

Reply:
Good catch, will keep it in mind next time!